Sources/CapacitorPluginTools/CapacitorPluginPackage.swift

```diff
@@ -2,19 +2,22 @@ import Foundation
 import CapacitorPluginSyntaxTools
 import JavascriptPackageTools
 
-enum CapacitorPluginError: Error {
+public enum CapacitorPluginError: Error {
     case objcFileCount(Int)
     case objcHeaderCount(Int)
     case oldPluginMissing
+    case updatePackageJSONFailure
```

Comment:
this error is never thrown, not sure if you forgot to throw it somewhere or if it can be deleted